eldritch/test_events.py

```diff
@@ -120,6 +148,123 @@ def testMoveMultipleSpaces(self):
     self.assertEqual(self.char.place.name, "Graveyard")
     self.assertEqual(self.char.movement_points, 1)
 
+  def testIllegalMoveMultipleSpaces(self):
+    self.char.movement_points = 1
+    movement = Movement(
+        self.char, [self.state.places[name] for name in ["Easttown", "Rivertown", "Graveyard"]])
+    self.assertFalse(movement.is_resolved())
+    self.assertEqual(self.char.movement_points, 1)
+
+    self.state.event_stack.append(movement)
+    self.resolve_until_done()
+
+    self.assertTrue(movement.is_resolved())
+    self.assertEqual(self.char.place.name, "Easttown")
+    self.assertEqual(self.char.movement_points, 0)
+
+  def testMoveMultipleThroughMonster(self):
+    self.char.fight_will_slider = 2
```

Comment:
Why did you change the fight_will_slider if you're going to have them pass the combat check anyway?